### ISSUE TITLE
Source suggestion feature

### ISSUE THREAD

**Issue by onurcakmakceng** [getanteon/gurubase#20]
It's nice to see sources of a guru. Besides there may be a button under the source list of a guru to take source suggestion from user.

**Comment by kursataktas** [comment]
Good idea. We'll think about this and update this thread once there is news.
